daemon/volumes_unix.go

```diff
@@ -97,6 +99,7 @@ func parseBindMount(spec string, mountLabel string, config *runconfig.Config) (*
 	return bind, nil
 }
 
+// sortMounts sorts an array of Mounts.
```

Comment:
sortMounts sorts an array of mounts in lexicographic order. This ensures that when mounting, the mounts don't shadow other mounts, e.g. mounting `/etc` and `/etc/resolv.conf`, `/etc/resolv.conf` must not be mounted first.